pkg/operator/staticpod/controllers.go

```diff
@@ -105,6 +105,7 @@ type Builder interface {
 	WithCustomInstaller(command []string, installerPodMutationFunc installer.InstallerPodMutationFunc) Builder
 	WithPruning(command []string, staticPodPrefix string) Builder
 	WithPodDisruptionBudgetGuard(operatorNamespace, operatorName, readyzPort, readyzEndpoint string, createConditionalFunc func() (bool, bool, error)) Builder
+	WithPodDisruptionBudgetGuardReadyZ(operatorNamespace string, operatorName string, createConditionalFunc func() (bool, bool, error), readyZConf common.ReadyZConf) Builder
```

Comment:
I don't think we should maintain both `WithPodDisruptionBudgetGuard` and `WithPodDisruptionBudgetGuardReadyZ`, but rather replace the original readyz args by ReadyZConf

Reply:
mentioned on slack, that's for backward compatibility given we're currently updating all services with library-go for the http/2 fix. I've done the change in the latest commit anyway, because people can also pick the prior commit hashes when updating.